docs/guide/testing.md

```diff
@@ -164,6 +164,17 @@ class ExampleComponentTest < ViewComponent::TestCase
 end
 ```
 
+## Testing with Devise
+
+Testing with Devise requires you include `Devise::Test::ControllerHelpers` in the `ViewComponent::TestCase` class. This
+is typically done by adding the following to `test/test_helper.rb`:
+
+```ruby
+class ViewComponent::TestCase
+  include Devise::Test::ControllerHelpers
+end
+```
+
```

Comment:
```suggestion
```

I don't think we should be prescriptive here. I think the Devise docs are clear enough on how to include the test helpers where you need them.

Reply:
@BlakeWilliams I merged those changes, however, I think it might be a mistake to not add the documentation. ViewComponent "strays" off of the Rails golden path. While Devise does describe how to setup tests, it doesn't specify how to set them up for `ViewComponent::TestCase`, which again is not a typical implementation.

While most engineers who google around would be able to find the actual solution, many will not, leading to potentially erroneous bug reports. By adding this do the documentation, it will help people self solve.

Reply:
> However, I think it might be a mistake to not add the documentation. ViewComponent "strays" off of the Rails golden path

I don't think we do, but I also don't feel super strongly about removing the documentation although it does feel a bit out-of-place.

Reply:
Happy to let it lie, and it can be re added if it becomes an annoyance. Looking deeper at the low number of people complaining it is probably a non issue.